### PR TITLE
#25224 integrity check 

### LINKED ISSUE #25224: PP - Integrity Checker, fixing conflicts is setting up identifier columns as null
### Parent Issue

_No response_

### Problem Statement

The asset_subtype, create_date, and owner fields is being set to null when fixing conflicts with the Integrity Checker.

### Steps to Reproduce

1. Start two dotCMS instances, and set the PP.
2. Publish a picture on the root folder of the sender instance and a different picture on the root folder of the receiver instance
3. Rename the file on the receiver to the same name as the one on the sender
4. Run the Integrity Checker.
5. It should generate a conflict.
6. Use the following query to verify that the asset_subtype is set: `select * from identifier where id = 'IDENTIFIER'`
7. Fix the conflict with IC.
8. Clean cache.
9. Run again the query to verify that the asset_subtype is set (actually the value is being set to null): `select * from identifier where id = 'IDENTIFIER'`

### Acceptance Criteria

The asset_subtype, create_date, and owner fields should not be set to NULL when the conflict is fixed.

### dotCMS Version

23.01 +

### Proposed Objective

Customer Support

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by AndreyDotcms**
### Proposed Changes
* The asset_subtype, create_date, and owner fields should not be set to NULL when the conflict is fixed.

### Checklist
- [x] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
**The main problem was found in FileAsset IC, but the problem was reproducible on Host IC. Folder IC and Page IC too.**

### CODE REVIEW

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityCheckerTest.java [hunk 1]
```diff
@@ -117,4 +120,48 @@ public void TestFixConflictGeneratesContentletAsJson() throws Exception {
...
+        final Contentlet contentlet = FileAssetDataGen.createFileAsset(folder, "text1FileAsset"+UUID.randomUUID().toString(), ".txt");
+        final FileAsset fileAsset = APILocator.getFileAssetAPI().fromContentlet(contentlet);
+        final ContentFileAssetIntegrityChecker integrityChecker = new ContentFileAssetIntegrityChecker();
+        final Tuple2<String, String> remoteIdentifierAndInode = introduceConflict(fileAsset ,endpointId.get());
+        final String remoteIdentifier = remoteIdentifierAndInode._1();
+        final String remoteWorkingInode = remoteIdentifierAndInode._2();
+
+        integrityChecker.executeFix(endpointId.get());
+
+        final DotConnect dotConnect = new DotConnect();
```
Comment (jdotcms) [inline dotCMS/src/integration-test/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityCheckerTest.java:142]: when you are using dotCOnnect on test, you should handle the connection in this case you have to close it at the end

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityCheckerTest.java (hunk 1 above)
Comment (AndreyDotcms) [reply dotCMS/src/integration-test/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityCheckerTest.java]: Added the requested changes https://github.com/dotCMS/core/pull/25325/commits/83fd7f33ff512c7e88b0f79d03bcda1bef3915c1

File: dotCMS/src/integration-test/java/com/dotcms/MainSuite1a.java [hunk 2]
```diff
@@ -88,7 +92,11 @@
         com.dotcms.content.elasticsearch.business.ESSiteSearchAPITest.class,
         com.dotcms.content.elasticsearch.business.ContentletIndexAPIImplTest.class,
         com.dotcms.content.elasticsearch.business.ES6UpgradeTest.class,
-        com.dotcms.content.elasticsearch.business.ESContentFactoryImplTest.class
+        com.dotcms.content.elasticsearch.business.ESContentFactoryImplTest.class,
```
Comment (nollymar) [inline dotCMS/src/integration-test/java/com/dotcms/MainSuite1a.java:95]: why did you add the existing tests on this class? were they missing in the others MainSuiteX.java files?

Comment (AndreyDotcms) [reply dotCMS/src/integration-test/java/com/dotcms/MainSuite1a.java:95]: I delete the existing ones and only leave the new one.

**Review by nollymar — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]
